### DISCUSSION TITLE
Change in Failed Login Response Property from error_description to msg in Supabase Auth

### DISCUSSION THREAD

**Discussion by sedatepace** [supabase/supabase#29404] [Questions]
It seems like the error_description property in the response for a failed login attempt using RestAPI with Auth has been changed to msg. When did this change occur?

Also, where can I check for announcements regarding such changes?

**Comment by silentworks** [comment]
You need to be more specific about which endpoint you called and show the actual payload you received. This would be a breaking change and would have been announced in the **Changelogs** section on here or it would be in the release notes of the https://github.com/supabase/auth repo.
